WordPress/src/main/java/org/wordpress/android/ui/pages/PageParentFragment.kt

```diff
@@ -70,12 +75,23 @@ class PageParentFragment : Fragment() {
         setHasOptionsMenu(true)
     }
 
+    @SuppressLint("RestrictedApi")
```

Comment:
I have deleted the lint suppress.

>  I also don't think usage of the setOptionalIconsVisible is necessary

But to be clear, are you saying we do not need to show the icons? Because by default, Android won't show it. 
Meanwhile, in EditPostSettingsFragment#showLocationPickerOrPopupMenu (line 1124), I see that we are forcing Android to show the icons.

Reply:
That was one more thing I wanted to mention. Why did you add the overflow menu at all? 
The `EditPostSettingsFragment#showLocationPickerOrPopupMenu` is different, that's not a menu in toolbar. The standard behaviour is to directly show the search icon in the toolbar (as it's done in all the other screens).